core/parentchain/light-client/src/light_validation.rs

```diff
@@ -163,12 +121,28 @@ where
 		let state_root = block_header.state_root();
 		Self::check_validator_set_proof(state_root, validator_set_proof, &validator_set)?;
 
+		self.initialize_relay(block_header, validator_set)
+	}
+
+	fn initialize_parachain_relay(
+		&mut self,
+		block_header: Block::Header,
+		validator_set: AuthorityList,
+	) -> Result<RelayId, Error> {
+		self.initialize_relay(block_header, validator_set)
+	}
+
+	fn initialize_relay(
+		&mut self,
+		block_header: Block::Header,
+		validator_set: AuthorityList,
+	) -> Result<RelayId, Error> {
```

Comment:
Is this method ever called from the outside? Or is it just a second layer for `initialize_parachain_relay` and `initialize_granpa_relay`? If so, remove this method from the trait and move it to the regular `impl` block